src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java

```diff
@@ -65,40 +138,45 @@ protected void processFiltered(File file, FileText fileText) {
     }
 
     /**
-     * Set the format of the regular expression to match.
+     * Setter to specify the format of the regular expression to match.
+     *
      * @param format the format of the regular expression to match.
      */
     public void setFormat(String format) {
         this.format = format;
     }
 
     /**
-     * Set the message to report for a match.
+     * Setter to specify the message which is used to notify about violations,
+     * if empty then default (hard-coded) message is used.
+     *
      * @param message the message to report for a match.
      */
     public void setMessage(String message) {
         this.message = message;
     }
 
     /**
-     * Set the minimum number of matches required per file.
-     * @param minimum the minimum number of matches required per file.
+     * Setter to specify the minimum number of matches required in each file.
+     *
+     * @param minimum the minimum number of matches required in each file.
      */
     public void setMinimum(int minimum) {
         this.minimum = minimum;
     }
 
     /**
-     * Set the maximum number of matches required per file.
-     * @param maximum the maximum number of matches required per file.
+     * Setter to specify the maximum number of matches required in each file.
+     *
+     * @param maximum the maximum number of matches required in each file.
      */
     public void setMaximum(int maximum) {
         this.maximum = maximum;
     }
 
     /**
-     * Set whether to ignore case when matching.
-     * @param ignoreCase whether to ignore case when matching.
+     *Setter to control whether to ignore case when searching.
```

Comment:
Missing a starting space.

Reply:
I am not seeing that in the discussion there. Whole issue is about where asterisk to aligned.
The issue here is the space after the asterisk, not the asterisk itself.

Reply:
You are right, there should be another `JavadocMissingWhitespaceCheck`.

Reply:
> Missing a starting space.

done

Reply:
JavadocMissingWhitespaceAfter AsteriskCheck

Reply:
@pbludov I am not seeing a new issue created for this.

Reply:
> @pbludov I am not seeing a new issue created for this.

Done: https://github.com/checkstyle/checkstyle/issues/7183

Reply:
Let's not block this PR by this off topic discussion

Reply:
@pbludov The code in this PR has still not been fixed.

Reply:
done